test/node-report/helper.js

```diff
@@ -0,0 +1,204 @@
+'use strict';
+require('../common');
+const child_process = require('child_process');
+const assert = require('assert');
+const fs = require('fs');
+const os = require('os');
+
+const osMap = {
+  'aix': 'AIX',
+  'darwin': 'Darwin',
+  'linux': 'Linux',
+  'sunos': 'SunOS',
+  'win32': 'Windows',
+};
+
+const REPORT_SECTIONS = [
+  'Node Report',
+  'JavaScript Stack Trace',
+  'JavaScript Heap',
+  'System Information'
+];
+
+const reNewline = '(?:\\r*\\n)';
+
+exports.findReports = (pid) => {
+  // Default filenames are of the form node-report.<date>.<time>.<pid>.<seq>.txt
+  const format = '^node-report\\.\\d+\\.\\d+\\.' + pid + '\\.\\d+\\.txt$';
+  const filePattern = new RegExp(format);
+  const files = fs.readdirSync('.');
+  return files.filter((file) => filePattern.test(file));
+};
+
+exports.isAIX = () => {
```

Comment:
Can you refactor this to use the `module.exports = {}` pattern... e.g.

```js
function isAIX() {
  return process.platform === 'aix';
}

module.exports = {
  isAIX
};
```

In general, however, why expose separate `is{Platform}` checks here? Are the checks in `common/index.js` not suitable?

Reply:
subsequent reviews and changes thereon made this change un-necessary.